tests/utils/abis.go

```diff
@@ -0,0 +1,72 @@
+package utils
+
+import (
+	"math/big"
+	"strings"
+
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/subnet-evm/accounts/abi"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/pkg/errors"
+
+	teleportermessenger "github.com/ava-labs/teleporter/abi-bindings/Teleporter/TeleporterMessenger"
+)
+
+type teleporterABI struct {
+	teleporterMessengerABI abi.ABI
+}
+
+func NewTeleporterABI() *teleporterABI {
+	parsed, err := abi.JSON(strings.NewReader(teleportermessenger.TeleporterMessengerABI))
```

Comment:
We should be able to call something like `teleportermessenger.TeleporterMessengerMetaData.GetAbi()` to get the same result here.